src/main/java/com/google/sps/data/Game.java

```diff
@@ -125,8 +125,8 @@ public Builder setNumTimesFinished(int val) {
         }
 
         /**
-        * Sets the number of star votes this game has received.
-        * @param val an int representing the number of times this game has been voted on.
+        * Sets the number of times this game's number of stars has been voted on.
+        * @param val an int representing the number of times this game's number of stars has been voted on.
         * @return a Builder object with the modified number of star votes.
         */
         public Builder setNumStarVotes(int val) {
```

Comment:
Rather than putting val, make the variable names more clear what they are for. I understand that there is only one variable so it doesn't make a difference, but for the API when people hover over the function and look at the parameters, having a more specific name will help since they won't see the implementation of it.

Reply:
Makes sense, I've done this for every time I had val. Thanks!